packages/spanner/system-test/spanner.js

```diff
@@ -2716,7 +2716,7 @@ var spanner = new Spanner(env);
           transaction.run('SELECT * FROM TxnTable', function(err, rows) {
             assert.ifError(err);
 
-            assert.strictEqual(rows.length, 2);
+            assert.strictEqual(rows.length, 1);
```

Comment:
Yeah, I've tried various combinations of running just the single test, just txn tests and running all tests and it passes every time.